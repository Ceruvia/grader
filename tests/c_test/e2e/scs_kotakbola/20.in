326 63 445
H 76
B 93
B 24
H 65
M 1
M 63
E
