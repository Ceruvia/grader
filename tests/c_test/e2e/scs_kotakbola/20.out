141 64 117
Tidak
